src/CommunityVoices/App/Api/Controller/Quote.php

```diff
@@ -143,6 +166,39 @@ protected function postQuoteUpload($request)
         );
     }
 
+    protected function postBatchDraft($request)
+    {
+        $files = $request->files->get('file');
+        if (sizeof($files) != 2) {
+            return [[],[],["item" => self::WRONG_NUM_FILES],[],[]]; // the third array returned to the api/view is errors with upload. This just lets the frontend know that there are errors.
+        }
+
+        // there may be a better way to do this, for now we are just relying on file names to indicate which document
+        foreach($request->files->get('file') as $file) {
+            if (str_contains(strtolower($file->getClientOriginalName()),"quote")) $quote = $file;
+            else if (str_contains(strtolower($file->getClientOriginalName()),"source")) $source = $file;
+        }
+        if (! (isset($source) && (isset($quote)))) return [[],[],["item" => self::WRONG_NAMES_FILES],[],[]];
```

Comment:
Explanation of why we are returning an array formatted like this would be nice.  I see why, but just in case it is not immediately clear.

Reply:
@Sammidysam See the comment on line 173. I can also add this to 181 if you would like?

Reply:
Ah, you are duplicating the generic error object.  Could you store it as a variable in the top of the function and put the comment there?

Reply:
Added extra comment